<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>toolGeneratePlacesWindow</class>
 <widget class="QMainWindow" name="toolGeneratePlacesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="boxGenerateMethod">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string>Случайно</string>
    </property>
    <item>
     <property name="text">
      <string>Случайно</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>По алфавиту</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>По алфавиту, в обратном порядке</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="textGenerateMethod">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Способ генерации</string>
    </property>
   </widget>
   <widget class="QLabel" name="textMaxKarts">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Максимальное количество картов</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="boxMaxKarts">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="textMinKarts">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Минимальное количество картов</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="boxMinKarts">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="textRunsPerParticipant">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество заездов участника</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="boxRunsPerParticipant">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGenerate">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Сгенерировать</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
